hack/provision.sh

```diff
@@ -101,6 +102,16 @@ install_actions_runner() {
   ./svc.sh install >/dev/null
   systemctl enable --now "$ACTIONS_RUNNER_SYSTEMD_UNIT"
 
+  if [ -f "/etc/systemd/system/$TRENTO_RUNNER_SYSTEMD_UNIT" ]; then
+    echo "  Warning: trento-runner systemd unit already installed. Removing..."
+    systemctl stop "$TRENTO_RUNNER_SYSTEMD_UNIT"
+    rm "/etc/systemd/system/$TRENTO_RUNNER_SYSTEMD_UNIT"
+  fi
+
+  echo "  Installing systemd unit"
+  ./svc.sh install >/dev/null
```

Comment:
Could we separate the bracket with a space after, like `./svc.sh install > /dev/null`?